src/js/edu-benefits/5490/config/form.js

```diff
@@ -93,53 +93,13 @@ const formConfig = {
     applicantInformation: {
       title: 'Applicant Information',
       pages: {
-        applicantInformation: applicantInformation(fullSchema5490, {
+        applicantInformation: applicantInformationPage(fullSchema5490, {
```

Comment:
We seem to be inconsistent across several pages. Some have page suffixes, some have create prefixes, or both/neither.